tests/scheduler/test_scheduler.py

```
@@ -634,6 +690,80 @@ def test_resume_normal(gcd_nop_project):
+    flow.node("stepthree", DummyTask())
+    flow.node("stepfour", DummyTask())
+    flow.edge("stepone", "steptwo")
+    flow.edge("steptwo", "stepthree")
+    flow.edge("stepthree", "stepfour")
+    project.set_flow(flow)
+
+    DummyTask.find_task(project).set("var", "check", "this")
+
+    assert project.run()
+    project.history("job0").get("record", "status", step="steptwo", index="0") == \
+        NodeStatus.SKIPPED
```

Ruff (0.14.4)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)